src/app/core/translations/pl_PL.ts

```diff
@@ -181,4 +181,5 @@ export default {
   CONFIRM_CANCEL_OFFER_NEW: 'Czy na pewno rezygnujesz z dodawania ogłoszenia?',
   CONFIRM_CANCEL_OFFER_EDIT: 'Czy na pewno rezygnujesz z edycji ogłoszenia?',
   CONFIRM_CANCEL_OFFER_DISC: 'Wprowadzone przez Ciebie informacje zostaną usunięte.',
+  LABEL_PHONE_COUNTRY_CODE: 'Telefoniczny kod kraju',
```

Comment:
@bbartosiak Chcemy to przez kogoś przepuścić? Musiałem jakiś tekst wpisać, bo to jest potem używane do obsługi błędów backendowych, więc wziąłem sformułowanie z wiki.